config/Common.xcconfig

```diff
@@ -1,4 +1,4 @@
-IPHONEOS_DEPLOYMENT_TARGET = 15.0
+IPHONEOS_DEPLOYMENT_TARGET = 16.0
```

Comment:
This change makes sense to me, but we might want to double-check with the Woo team that they agree with bumping the deployment target of the whole project as part of this PR.